app/src/test/kotlin/com/exallium/todoapp/screenbundle/ScreenBundleHelperImplTest.kt

```diff
@@ -19,10 +19,12 @@ class ScreenBundleHelperImplTest {
 
     companion object {
         private val TITLE = "screenBundleHelper.title"
+        private val NOTE_ID = "screenBundleHelper.noteId"
 
         private val APP_NAME = "TodoApp"
         private val TEST_TITLE_RES_ID = 1
         private val TEST_TITLE_STRING = "test title"
+        private val TEST_NOTE_ID_STRING = "test note id"
```

Comment:
please check my refactor. my understanding is const keyword would force me to declare these above class declaration which I don't think we want to do.